smac/tae/execute_func.py

```diff
@@ -79,6 +79,9 @@ def run(self, config, instance=None,
         arguments = {'logger': logging.getLogger("pynisher"),
                      'wall_time_in_s': cutoff,
                      'mem_in_mb': memory_limit}
+        
+        if cutoff is not None and cutoff > MAX_CUTOFF:
+            self.logger.warn("Pynisher may only limit the cutoff for up to %d" %(MAX_CUTOFF))
```

Comment:
I see. But we should avoid any behaviour that might differ too much on different systems. Therefore, we should either check the limit dynamically, or fail if a user tries to hit the bar